WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -47,11 +47,12 @@ private extension ItemListView {
                             .font(.posTitleRegular)
                     })
                     .foregroundColor(.posPrimaryTexti3)
+                    .padding(.trailing, Constants.headerPadding)
                 }
             }
             if viewModel.shouldShowHeaderBanner {
                 bannerCardView
-                    .padding(.vertical, 16)
+                    .padding(.vertical, Constants.headerPadding)
```

Comment:
Again, this is perhaps a semantically different thing than the padding needed for the top of the header (to align the Products and Cart titles.

I get that the name could refer to any padding around the header, but I think that only makes sense if we apply it to all the edges of all the headers, and we can't do that.

Reply:
I see what you mean, I've separated and renamed these for specificity here: https://github.com/woocommerce/woocommerce-ios/pull/13707/commits/cd875bc6c557378a721c9062c6875dd51fecba55